test_scripts/Resumption/InteriorVehicleData/common_resumptionsInteriorVD.lua

```diff
@@ -0,0 +1,401 @@
+---------------------------------------------------------------------------------------------------
+-- Common module
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local actions = require("user_modules/sequences/actions")
+local utils = require("user_modules/utils")
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local json = require("modules/json")
+local test = require("user_modules/dummy_connecttest")
+local events = require('events')
+
+--[[ General configuration parameters ]]
+config.defaultProtocolVersion = 2
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application2.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+config.application2.registerAppInterfaceParams.isMediaApplication = false
+
+--[[ Variables ]]
+local m = actions
+m.modules = { "RADIO", "CLIMATE", "SEAT", "AUDIO", "LIGHT", "HMI_SETTINGS" }
+m.hashId = {}
+m.cloneTable = utils.cloneTable
+m.wait = utils.wait
+m.tableToString = utils.tableToString
+
+-- [[ Common Functions ]]
+function  m.GetInteriorVehicleData(pModuleType, isSubscribe, hmiReqExpectTimes, hashChangeExpectTimes, pAppId)
+  if not pAppId then pAppId = 1 end
+
+  local requestParams = {
+    moduleType = pModuleType,
+    subscribe = isSubscribe
+  }
+
+  local cid = m.getMobileSession(pAppId):SendRPC("GetInteriorVehicleData",requestParams)
+  EXPECT_HMICALL("RC.GetInteriorVehicleData", requestParams)
+  :Do(function(_, data)
+      m.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+        { moduleData = m.getModuleControlData(pModuleType), isSubscribed = isSubscribe })
+    end)
+  :Times(hmiReqExpectTimes)
+  m.getMobileSession(pAppId):ExpectResponse(cid,
+    { success = true, resultCode = "SUCCESS", moduleData = m.getModuleControlData(pModuleType), isSubscribed = isSubscribe})
+
+  m.getMobileSession(pAppId):ExpectNotification("OnHashChange")
+  :Do(function(_,data)
+      m.hashId[pAppId] = data.payload.hashId
```

Comment:
@GetmanetsIrina  `data.payload.hashId` should be changed to `data.payload.hashID`